src/core/processing/qgsprocessingcontext.h

```diff
@@ -90,10 +90,10 @@ class CORE_EXPORT QgsProcessingContext
     void setExpressionContext( const QgsExpressionContext &context ) { mExpressionContext = context; }
 
     /**
-     * Returns a reference to the project used for storing temporary layers during
+     * Returns a reference to the layer store used for storing temporary layers during
      * algorithm execution.
      */
-    QgsProject &temporaryLayerStore() { return tempProject; }
+    QgsMapLayerStore &temporaryLayerStore() { return tempLayerStore; }
```

Comment:
Is there a reason for returning this by ref rather than ptr? Would be nice to have it aligned with `QgsProject`. Myself I always pass QObjects by pointer as a rule (and thought that everyone would do it ;) )